docs/faq/general.md

```
@@ -0,0 +1,53 @@
+
+- disable fast boot within the BIOS
+- disable fast boot in Windows
+- always fully shutdown after using and switchting to another OS so the hardware gets correctly initialized
+
+If you still experience an issue, hold down the power button while on battery for a few seconds to force a shutdown. This often helps to reset some things.
+
+These steps are not needed if you are running Linux exclusively.
+
+### Note for ROG Flow X13 (2021)
+
+BIOS versions 408 & 409 cannot boot a Linux kernel newer than 5.15.x so you will need to upgrade to the 410 bios [here](https://rog.asus.com/laptops/rog-flow/2021-rog-flow-x13-series/helpdesk_bios).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)